docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -146,7 +146,152 @@ The file to export the slashing protection database to.
 
 Exports the database in the [validator client interchange format] format.
 
+## `repair`
+
+Repairs corrupted slashing-protection data files used by Teku.
+
+### check-only-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled=false
+    ```
+
+Reads and reports potential slashing-protection file problems, but doesn't update any files.
+You can specify which files are checked using [`--config-file`](#config-file_2), [`--data-base-path`](#data-base-path-data-path),
+or [`--network`](#network).
```

Comment:
network won't help find files, as this implies, but the first 2 will.

network is in this command so that we can get the genesis time for the specified network if we need to calculate a slot...